src/main/1.3/scala/com/holdenkarau/spark/testing/StreamingActionBase.scala

```diff
@@ -107,15 +107,18 @@ trait StreamingActionBase extends StreamingSuiteBase {
       val startTime = System.currentTimeMillis()
       while (batchCountListener.batchCount < numBatches &&
         System.currentTimeMillis() - startTime < maxWaitTimeMillis) {
+        // TODO What if we need more time than maxWaitTimeMillis
```

Comment:
Most tests should finish in under 10000ms but if the user needs more time they can just override it.